scripts/process.zs

```diff
@@ -276,6 +276,12 @@ function beneficiate(
     crush(input, dustOrGem, exceptions ~ 'macerator thermalCentrifuge crushingBlock', extraList, extraChances, { bonusType: 'MULTIPLY_OUTPUT' });
   }
 
+  // Mass Spectrometer Dirty Gem processing
+  val gem = utils.getSomething(oreName, ['gem', 'any'], amount + step);
+  if (!isNull(gem)) {
+    workEx("massspectrometer", exceptions, [input], null, [output], null, null, null, null);
```

Comment:
Why you changed letter case here? You dont need to do that, since only case in `workEx` function matter. There you can use any case. Revert back.
Also, you need to keep `opts` parameter for consistensy, even if it not used.